pkg/storage/storage.go

```diff
@@ -84,3 +94,15 @@ func SetAsyncStorage(storageEngine Storage) error {
 	storageAsync = storageEngine
 	return nil
 }
+
+func isStronglyConsistent(ctx xcontext.Context) bool {
+	value := ctx.Value("storage_consistency_model")
```

Comment:
should this key be a constant?

Reply:
yeah, that's why I was asking for comments. This magic value seems rather out of place. I'll extract it to a "WithValue" kind of thing for the context, but it will live in storage layer